components/Login/index.js

```diff
@@ -0,0 +1,115 @@
+import { signIn, signOut, useSession } from "next-auth/react";
+import styled from "styled-components";
+import Image from "next/image";
+import { useState } from "react";
+
```

Comment:
There are some improvements I can propose to your code : 
2. Add PropTypes for type checking: This helps to avoid bugs by ensuring the correct prop types are used

Make therefore an import here : 
import PropTypes from "prop-types";